packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,98 @@
+.navMenuContainer {
+    --hoverOffset: var(--space-smallest);
+    --negativeHoverOffset: calc(-1 * var(--hoverOffset));
+    --subItemInset: calc(var(--space-small) + var(--space-smaller));
+    --navItemHeight: 40px;
+    display: flex;
+    flex-direction: column;
+    padding: 36px 0 0 0;
+    height: 100dvh;
+    background-color: var(--color-surface--background);
+    width: 200px;
```

Comment:
Wider than I'd originally mocked up - the 188 was referencing Jobber's sidenav, but we don't necessarily need to be married to that value and this solves a great deal of the wrapping issues